fendermint/vm/topdown/examples/run_polling.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+use async_stm::atomically_or_err;
+use clap::Parser;
+use fendermint_vm_topdown::{
+    Config, Error, IPCAgentProxy, IPCParentFinality, InMemoryFinalityProvider,
+    ParentFinalityProvider, ParentViewProvider, PollingParentSyncer,
+};
+use fvm_shared::address::{set_current_network, Network};
+use ipc_agent_sdk::apis::IpcAgentClient;
+use ipc_agent_sdk::jsonrpc::JsonRpcClientImpl;
+use ipc_sdk::subnet_id::SubnetID;
+use num_traits::FromPrimitive;
+use std::str::FromStr;
+use std::sync::Arc;
+use std::time::Duration;
+use tracing::Level;
+
+#[derive(Parser, Debug)]
+pub struct Options {
+    /// The URL of the ipc agent's RPC endpoint.
```

Comment:
Ah, okay, I figured later this would be the agent. 

You know what would be super useful is to add a README.md to the `topdown` crate with a section about how to run this example, which should include instructions for starting both Lotus and the agent, or at least links to where this is explained, because I don't think readers will know, which limits the usefulness of this example.